dev-util/capstone/capstone-4.0.1.recipe

```diff
@@ -65,3 +71,10 @@ INSTALL()
 	fixPkgconfig
 	packageEntries devel $developDir
 }
+
+TEST()
+{
+	wget ${fuzz_test_url}/raw/master/corpus/${fuzz_test_file}
+	unzip -o -q ${fuzz_test_file} -d suite/fuzz	
```

Comment:
```diff
-	wget ${fuzz_test_url}/raw/master/corpus/${fuzz_test_file}
-	unzip -o -q ${fuzz_test_file} -d suite/fuzz	
+	fuzz_test_file="corpus-libFuzzer-capstone_fuzz_disasmnext-latest.zip"
+	unzip -o -q $sourceDir2/capstonefuzz-$srcGitRev/corpus/${fuzz_test_file} -d suite/fuzz 
```

Reply:
Is there a way we could use an "if" "fi" argument for these (so it doesn't unzip on every test or errors out on being already there)?
Or maybe move this into PATCH()?

Reply:
@Begasus I've added a check to see if the folder exists or not to prevent it unzipping every-time TEST() is run.